tests/e2e/vLLM/test_vllm.py

```diff
@@ -146,3 +163,35 @@ def test_vllm(self):
     def tear_down(self):
         if self.save_dir is not None:
             shutil.rmtree(self.save_dir)
+
+    def _check_session_contains_recipe(self) -> None:
+        session = active_session()
+        recipe_yaml_str = session.get_serialized_recipe()
+        assert recipe_yaml_str is not None
+
+    def _check_save_dir_has_expected_files(self):
```

Comment:
move these to utils

Reply:
YOu only this func here, its not shared func. No need to move it